ml-agents/mlagents/trainers/agent_processor.py

```diff
@@ -144,7 +144,7 @@ def add_experiences(
                     )
                     for traj_queue in self.trajectory_queues:
                         traj_queue.put(trajectory)
-                    self.experience_buffers[global_id] = []
+                    del self.experience_buffers[global_id]
```

Comment:
Need to del `last_step_result` and `last_take_action_outputs` as well. Probably can do it right after the experience buffer `del`

Reply:
hmmm, unfortunately, it seems last_step_result as well as policy.previous_actions are modified after the check for done. (So even if I delete them, they will be re-added). I need to do more experiments...